src/lsst/cm/tools/core/error_code_decisions.yaml

```diff
@@ -136,6 +145,25 @@ pandaErrorCode:
             rescue: 1
             panda: 1
             intensity: 0
+        p_fgcm_380:
```

Comment:
I was thinking it would be a good idea to keep a record of them in case we see them again, so that we know. My philosophy with this file was that we record as much as possible (so as to properly answer questions like "have we ever seen this before?" and "what should we do if we see something like this?") It should also include information on whether the errors are campaign-stopping. The field "panda" currently records whether an error is a panda issue rather than a pipelines issue, but with DM-38007 it will be a status enum which should include pipelines, panda and infrastructure errors.

For the rest of the fields, as is currently:
**function**: which function did the error occur when we saw it?
**known**: have we seen this before?/do we know what it is? [if 0, "ticket" should be empty]
**ticket**: which Jira tickets are associated with this issue?
**resolved**: do we know what to do with it?/if it's a bug, did we fix it? [if this field is True and we see an error come up again that is not also marked "rescue" we know that the issue is serious]
**rescue**: can we solve this issue with a rescue?

Between these fields I think we can figure out what to do (stop the campaign, run a rescue, nothing, ...etc which we called the [ErrorAction](https://github.com/lsst-dm/cm_tools/blob/bcc0a7bbbedfa9b1f7c45e00978b1c965a3752a0/src/lsst/cm/tools/db/error_table.py#L19)) but I'm open to designing this to be better with noting and looking out for campaign-stopping errors also. We could also put an Action in as an optional field in the error table. If you like we can throw this discussion/work in with DM-38007.